portfolio/src/main/webapp/index.html

```diff
@@ -115,22 +115,37 @@ <h1 class="sub-message">Where you pay to stay</h1>
     <div id="map"></div>
     <div class="division-bar"></div>
     <div class="division-bar"></div>
-    <form action="/data" class="contact-form" method="post">
-      <p>Say Hi!</p>
-      <input class="form-name" type="text" name="name" placeholder="Name">
-      <input class="form-comment" type="text" name="comment" placeholder="Enter Comment">
-      <button class="form-button" type="submit" name="button">Send</button>
-    </form>
-
     <div id="images-container">
-      <form id="image-upload-form" class="hidden" method="POST" enctype="multipart/form-data">
-        <h1>Upload an image:</h1>
-        <input type="file" name="image">
-        <br /><br />
-        <button type="submit">Submit</button>
-      </form>
-    </div>
+      <div class="images-title">
+        <h1>Try Google's Vision API</h1>
+        <p>keep it PG...</p>
+      </div>
+      <div class="image-presenter"></div>
+      <div class="division-bar"></div>
 
+      <div class="image-uploader">
+        <div class="labels-container"></div>
+        <div class="image-form-container">
+          <form id="image-upload-form" class="hidden" method="POST" enctype="multipart/form-data">
+            <div class="custom-file file-picker">
+              <input class="file-upload" name="image" type="file" class="custom-file-input" id="customFile">
```

Comment:
keep these uniform, `id="custom-file"`

Reply:
got it! It was a mistake and i removed it

Reply:
unfortunately this id is part of bootstrap so if I change it, my file input will not work